integration_tests/test_str_01.py

```diff
@@ -33,10 +33,16 @@ def test_str_slice():
     assert a[2:4] == "23"
     assert a[2:3] == a[2]
 
+def test_str_repeat():
+    a: str
+    a = "Xyz"
+    assert a*3 == "XyzXyzXyz"
+
```

Comment:
Thanks for the PR! :) Can we add these:
```diff
diff --git a/integration_tests/test_str_01.py b/integration_tests/test_str_01.py
index 348fdc66a..39e4ba91a 100644
--- a/integration_tests/test_str_01.py
+++ b/integration_tests/test_str_01.py
@@ -37,6 +37,8 @@ def test_str_repeat():
     a: str
     a = "Xyz"
     assert a*3 == "XyzXyzXyz"
+    assert 3*a*3 == "XyzXyzXyzXyzXyzXyzXyzXyzXyz"
+    assert a*-1 == ""
 
 def check():
     f()
```
Again, the string comparison doesn't work properly yet, so we should work on that too.

Reply:
Good catch, we need to cover those cases as well. Thanks for the review!